### PR TITLE
Feature/implement alternative db storage

### PR DESCRIPTION
**Pull request by skraynick**

### CODE REVIEW

File: android-sdk/build.gradle [hunk 1]
```diff
@@ -22,6 +22,9 @@ dependencies {
 
     compile 'net.danlew:android.joda:2.7.2'
 
+    compile fileTree(dir: 'libs', include: ['*.jar'])
+    compile files('libs/sensorberg_orm-0.7.1.201405082137.jar')
```
Comment (mrsasha) [inline android-sdk/build.gradle]: I would add jitpack.io integration here instead of having a jar.

**Comment by mrsasha** [comment]
Merge conflicts need to be resolved. There are lots of failing tests, don't know how many of those will get fixed when we rebase.

App build seems to be working ok. Should we merge now and fix the tests later or do we go for the 100% now? @falkorichter @skraynick

**Comment by skraynick** [comment]
@mrsasha I have 3 failed tests. Investigating to see if related to changes. Some of the tests might not work because of the issue of generation of the table columns by the ORM. When I run the tests and put in premade db instance with the necessary columns I only get three fails, which on the surface do not look like the are related to the changes.

File: android-sdk/build.gradle (hunk 1 above)
Comment (skraynick) [inline android-sdk/build.gradle]: Will do. Either way I should see if I now have the ability to write to our repo.. so I can get the fork off my private/public personal repo.

**Comment by skraynick** [comment]
@mrsasha I added a test util class that will create the table columns so that should take care of the failed tests for the most part. There is one thing that remains problematic with the branch. There is a mockito error. Perhaps on Monday you can take a peak. I will see if I can figure out over the next few days when I get a chance... Otherwise everything is much better.  Conflicts are also dealt with.  Will move over the sugarorm stuff today as well.

**Comment by mrsasha** [comment]
cool, that's good news. should we merge it then?

**Comment by skraynick** [comment]
@mrsasha no. Just switching over the library.

**Comment by skraynick** [comment]
@mrsasha go ahead and merge. I moved over the sugar library over to sensorberg (github). There are two branchs... sensorbergORM and the main.  The main I think would be nice to leave as a tie to the sugar project itself.  I reconsidered your approach. I like it, if we wanted our changes to be public, but if we make changes to the wider open source project we can always make them and then create a pull... unless you still want to host it else where.  I leave it up to you since you are the reviewer. ;)

**Pull request merged**